### PR TITLE
Update changelog with 0.14 changes

### PR DESCRIPTION
**Pull request by philipnrmn**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -11,9 +11,29 @@ release.
...
+In 0.14.0 we replace the filter with a search which leads to a detailed search result view. There are
+various improvements in the search interaction, including the user being returned to their former group
+context after the search term has been cleared.
+
+#### Better health status feedback
+The health bars in the application list view are now provided in the application detail view. The tooltips
+in the overview have been consolidated to make them easier to understand. Additionally, the styles of the
+deployment table have been updated inline with the applications list view.
+
+#### IP-per-container UI
```
Comment (gkleiman) [inline changelog.md]: We're calling this IP-per-task in Marathon.

**Comment by gkleiman** [comment]
Thanks a lot, just a minor comment!

**Comment by philipnrmn** [comment]
Thanks for the review - I fixed that up.

**Comment by gkleiman** [comment]
:rocket: thanks!

**Pull request merged**